src/main/kotlin/io/sip3/salto/ce/rtpr/RtprHandler.kt

```diff
@@ -192,6 +197,11 @@ open class RtprHandler : AbstractVerticle() {
     }
 
     open fun handle(packet: Packet, report: RtpReportPayload) {
+        if (!validate(packet, report)) {
+            logger.debug { "Handle invalid RTP Report. Report: ${JsonObject.mapFrom(report)}" }
```

Comment:
"Handling invalid RTCP report"?

Reply:
I decided to validate RTP Reports instead RTCP. We have not enough data to do validation with RTCP Streams or raw Sender Reports